Exiting with code=1
internal/modules/cjs/loader.js:*
    internalBinding('errors').triggerUncaughtException(
                              ^

AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:

1 !== 2

    at Object.<anonymous> (*test*message*error_exit.js:*:*)
    at Module._compile (internal/modules/cjs/loader.js:*:*)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:*:*)
    at Module.load (internal/modules/cjs/loader.js:*:*)
    at Function.Module._load (internal/modules/cjs/loader.js:*:*)
    at ModuleWrap.<anonymous> (internal/modules/esm/translators.js:*:*)
    at ModuleJob.run (internal/modules/esm/module_job.js:*:*)
    at async Loader.import (internal/modules/esm/loader.js:*:*) {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: 1,
  expected: 2,
  operator: 'strictEqual'
}
